test/integration/commands/xmlupload/test_resource_creation.py

```diff
@@ -86,8 +86,10 @@ def test_one_resource_without_links(ingest_client_mock: AssetClient) -> None:
         "route": "/v2/resources",
         "data": {
             "@type": "http://0.0.0.0:3333/ontology/9999/onto/v2#foo_1_type",
-            "rdfs:label": "foo_1_label",
-            "knora-api:attachedToProject": {"@id": "https://admin.test.dasch.swiss/project/MsOaiQkcQ7-QPxsYBKckfQ"},
+            "http://www.w3.org/2000/01/rdf-schema#label": "foo_1_label",
+            "http://api.knora.org/ontology/knora-api/v2#attachedToProject": {
+                "@id": "https://admin.test.dasch.swiss/project/MsOaiQkcQ7-QPxsYBKckfQ"
```

Comment:
not part of this PR, but is this a correct project IRI? It looks outdated.

Reply:
I have never taken a look. So it could be an outdated project iri...